esf_main.tex

```diff
@@ -806,11 +822,12 @@ \subsection{LV part 2}
 \section{Overall Grounding Concept}
 \subsection{Description of the Grounding Concept}
 %Describe how you intend to achieve the resistances between components at the required levels as defined in EV4.3.
-TODO
+The vehicle's steel chassis is uses as GLVS ground. This ground will be established inside the master switch panel enclosure by attachment to a piositively-retained ring terminal on a chassis member that has been locally stripped of paint. All components of the GLVS will be grounded to this point on the chassis through the wiring harness for simplicity of installation and mechanical robustness. All conductive mechanical enlosures are mounted to the chassis with conductive metal fasteners. Electrical connection of the suspension members and knuckles to the main vehicle frame will be achieved by short jumpers with ring terminals on either end in the suspension member bolt stackup on either side of the spherical bearing.
```

Comment:
> "The vehicle's steel chassis is **used** as GLVS ground."
Quick typo fix.

We might also want to have a grounding point at the front of the chassis as well, just to ensure that no part of the chassis floats above ground.